fineract-provider/gradle.properties

```diff
@@ -16,5 +16,5 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-releaseVersion=1.0.0
+releaseVersion=1.4.0
 buildType=BUILD
```

Comment:
Here we have the version hardcoded - for the Swagger documentation we pick it automatically from the git describe output. We should probably make this consistent and use one or the other...

Reply:
I saw that you use a Gradle version plugin to do this and I actually prefer that way. For this release I'll leave it to the hard coded version number, but would suggest your method for the whole project. Note: the version of the client JAR in the release branch is not correct (shows something like "client-1.3.0-xyz-dirty.jar"; I do understand that it creates that based on the last release tag; we just have to make sure that this works correctly when the release build is running (release tag 1.4.0 has to exist).

Reply:
Yes - good point, please let's make sure it works correctly when you do the release build. If not happy to help fix it if needed.